src/enigma2/Epg.cpp

```diff
@@ -40,30 +40,31 @@ bool Epg::Initialise(enigma2::Channels &channels, enigma2::ChannelGroups &channe
   //add an initial EPG data per channel uId, sref and initial EPG
   for (auto& channel : channels.GetChannelsList())
   {
-    EpgChannel newEpgChannel;
+    std::shared_ptr<data::EpgChannel> newEpgChannel = std::make_shared<data::EpgChannel>();
```

Comment:
If you only want to call the default ctor, std::make_shared does not make to much sense. A simple `std::shared_ptr<EpgChannel> newEpgChannel;` does exactly the same.

Reply:
And again